### PR TITLE
added ember apps on the javascript-with-nodejs

### PR DESCRIPTION
**Pull request by sumn2u**
added ember apps and travis.yml script which show how to deploy ember app and automate test using travis CI

### CODE REVIEW

**Comment by MariadeAnton** [comment]
@sumn2u this is great, thank you!

File: user/languages/javascript-with-nodejs.md [hunk 1]
```diff
@@ -121,6 +121,51 @@ npm install
 
 Note that `npm install` can fail if a shrinkwrapped git dependency pointing to a branch has its HEAD changed.
 
+## Ember Apps
+You can build your Ember Apps on Travis CI and can automate test. By default ember comes with [`Qunit`](http://qunitjs.com/) as testing framework
+
+```yaml
+dist: trusty
```
Comment (MariadeAnton) [inline user/languages/javascript-with-nodejs.md]: can you add a `sudo: required` line right above `dist: trusty` ? That's a recommended practice per https://docs.travis-ci.com/user/trusty-ci-environment to route builds to Trusty :)

File: user/languages/javascript-with-nodejs.md [hunk 2]
```diff
@@ -121,6 +121,51 @@ npm install
...
+addons:
+  apt:
+    sources:
+      - google-chrome
+    packages:
+      - google-chrome-stable
+language: node_js
+node_js:
+  - "0.12"
+env:
```
Comment (MariadeAnton) [inline user/languages/javascript-with-nodejs.md]: would be worth adding a note here mentioning that this is an example of the different ember releases you can run within your build?

File: user/languages/javascript-with-nodejs.md [hunk 3]
```diff
@@ -121,6 +121,51 @@ npm install
...
+    - EMBER_VERSION=default
+    # - EMBER_VERSION=release
+    # - EMBER_VERSION=beta
+    # - EMBER_VERSION=canary
+matrix:
```
Comment (MariadeAnton) [inline user/languages/javascript-with-nodejs.md]: is this a common practice?

File: user/languages/javascript-with-nodejs.md [hunk 4]
```diff
@@ -121,6 +121,51 @@ npm install
...
+  fast_finish: true
+  allow_failures:
+    - env: EMBER_VERSION=release
+    # - env: EMBER_VERSION=beta
+    # - env: EMBER_VERSION=canary
+
+before_install:
+    - export PATH=/usr/local/phantomjs-2.0.0/bin:$PATH
```
Comment (MariadeAnton) [inline user/languages/javascript-with-nodejs.md]: a comment before this line:

# setting the path for phantom.js 2.0.0

File: user/languages/javascript-with-nodejs.md [hunk 5]
```diff
@@ -121,6 +121,51 @@ npm install
...
+    - export DISPLAY=:99.0
```
Comment (MariadeAnton) [inline user/languages/javascript-with-nodejs.md]: same kind of comments on each specific setting?

For example:
 `# starting a GUI to run tests, per https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI` or `# upgrade npm to 2.0` ...

What do you think?

File: user/languages/javascript-with-nodejs.md [hunk 6]
```diff
@@ -140,6 +185,7 @@ env:
 
 More info on [testing against laika](https://github.com/arunoda/travis-ci-laika).
 
+
```
Comment (MariadeAnton) [inline user/languages/javascript-with-nodejs.md]: could you remove this blank line? Thank you!

File: user/languages/javascript-with-nodejs.md (hunk 1 above)
Comment (sumn2u) [inline user/languages/javascript-with-nodejs.md]: sure .

File: user/languages/javascript-with-nodejs.md (hunk 2 above)
Comment (sumn2u) [inline user/languages/javascript-with-nodejs.md]: :+1:

File: user/languages/javascript-with-nodejs.md (hunk 3 above)
Comment (sumn2u) [inline user/languages/javascript-with-nodejs.md]: we can run ember in different releases such as canary , beta , release etc. here we set the version as default which will set the latest stable version.

File: user/languages/javascript-with-nodejs.md (hunk 6 above)
Comment (sumn2u) [inline user/languages/javascript-with-nodejs.md]: :+1:

File: user/languages/javascript-with-nodejs.md (hunk 5 above)
Comment (sumn2u) [inline user/languages/javascript-with-nodejs.md]: ya its a better idea we can set comment over here .

**Pull request closed without merging**